reference/factorization/cholesky_kernels.cpp

```diff
@@ -204,8 +204,15 @@ void factorize(std::shared_ptr<const DefaultExecutor> exec,
                 const auto col = cols[dep_nz];
                 if (col < row) {
                     const auto val = vals[dep_nz];
-                    const auto nz = row_begin + lookup.lookup_unsafe(col);
-                    vals[nz] -= scale * val;
+                    if (!full_fillin) {
```

Comment:
if constexpr

Reply:
this is not template, so I can not use constexpr